components/OrchestratorList/index.tsx

```diff
@@ -400,7 +400,7 @@ const OrchestratorList = ({ data, protocolData, pageSize = 10 }) => {
                       </Flex>
                       <Link
                         passHref
-                        href="https://github.com/livepeer/explorer/blob/main/ROI.md"
+                        href="https://docs.livepeer.org/delegators/how-to-guides/yield-calculation"
```

Comment:
Let's put this doc under a "Reference" category. The url should be `https://docs.livepeer.org/delegators/how-to-reference/yield-calculation`

Looks like existing category needs to be renamed from "Reference" to "Guides".  
<img width="296" alt="image" src="https://user-images.githubusercontent.com/555740/173429484-1603dca8-60bd-4693-a184-88261aa9ec3e.png">